### PR TITLE
Inpaint module

### PR DESCRIPTION
**Pull request by xiaoweiw-nv**
Add image inpaint module using the [MAT](https://github.com/fenglinglwb/mat) model.

The inference is run by PyTorch, I tried to export the model to onnx but the implementation contains several plugins so onnx does not work. I literally wrote the module in an afternoon : ) This module should be a good demo to show how fast the developer can integrate a PyTorch model into BMF.

### CODE REVIEW

File: bmf/demo/pytorch_module/test_torch_module.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import sys
+
+sys.path.append("../../")
+import bmf
+
+sys.path.pop()
+
+def test():
+    input_video_path = "./test1.png"
```
Comment (sfeiwong) [inline bmf/demo/pytorch_module/test_torch_module.py]: 要不改成bmf自带测试文件中的blue.png或overlay.png ?

Comment (xiaoweiw-nv) [reply bmf/demo/pytorch_module/test_torch_module.py]: 可以的，这两张图片的路径在哪里呀？我搜了下没搜到

Comment (sfeiwong) [reply bmf/demo/pytorch_module/test_torch_module.py]: 和其他物料打包放到这里了：https://github.com/BabitMF/bmf/releases/download/files/files.tar.gz

Comment (xiaoweiw-nv) [reply bmf/demo/pytorch_module/test_torch_module.py]: 已更新

**Review by sfeiwong — APPROVED** [review]

**Review by sfeiwong — APPROVED** [review]

**Pull request merged**